app/models/spree/payment_method.rb

```diff
@@ -110,23 +110,9 @@ def has_distributor?(distributor)
     end
 
     def self.clean_name
-      case name
-      when "Spree::PaymentMethod::Check"
-        "Cash/EFT/etc. (payments for which automatic validation is not required)"
-      when "Spree::Gateway::Migs"
-        "MasterCard Internet Gateway Service (MIGS)"
-      when "Spree::Gateway::Pin"
-        "Pin Payments"
-      when "Spree::Gateway::StripeConnect"
-        "Stripe"
-      when "Spree::Gateway::StripeSCA"
-        "Stripe SCA"
-      when "Spree::Gateway::PayPalExpress"
-        "PayPal Express"
-      else
-        i = name.rindex('::') + 2
-        name[i..-1]
-      end
+      i = name.rindex('::') + 2
+      i18n_name = "spree.admin.payment_methods.providers." + name[i..-1].downcase
+      I18n.t(i18n_name)
```

Comment:
I'm not a big fan of these single-letter variables and arbitrary numbers. Could we use something a bit more readable here? Maybe something like: 
```ruby
I18n_key = "spree.admin.payment_methods.providers." + name.demodulize.downcase
I18n.t(I18n_key)
```
https://apidock.com/rails/v4.2.1/String/demodulize

Reply:
Yeah, that seems a lot more readable. I'm changing this now

Reply:
Done @Matt-Yorkley

Reply:
That looks much nicer now, and the specs are still passing :ok_hand:

Reply:
Agree, that's a nice cleanup.